less/cards.less

```diff
@@ -625,15 +644,14 @@
           width:  150px;
         }
         .area  {
```

Comment:
why not removing the duplicated space before the bracket in the meantime? :wink:

Reply:
fix